libs/utils/address.go

```diff
@@ -37,9 +52,5 @@ func ValidateAddr(addr string) (string, error) {
 		return addr, nil
 	}
 
-	resolved, err := net.ResolveIPAddr("ip4", addr)
-	if err != nil {
-		return addr, err
-	}
-	return resolved.String(), nil
+	return addr, nil
```

Comment:
Looks like this and `return` above are identical. Probably we should return error in `return` above.

Also, we can use https://pkg.go.dev/net/netip#ParseAddr to get error message, `net.ParseIP` is an old and not friendly way to parse ip.